examples/arbitrarycallbackdatabot.py

```diff
@@ -84,7 +84,7 @@ def handle_invalid_button(update: Update, context: CallbackContext) -> None:
 def main() -> None:
     """Run the bot."""
     # We use persistence to demonstrate how buttons can still work after the bot was restarted
-    persistence = PicklePersistence(filename='arbitrarycallbackdatabot.pickle')
+    persistence = PicklePersistence(filepath='arbitrarycallbackdatabot.pickle')
```

Comment:
@Bibo-Joshi shouldn't this not have a file extension? It can cause problems when `single_file` is `False`. The other example (`persistentconversationbot.py`) doesn't have this.

Reply:
yeah, maybe that's better. I'm rather sure that it doesn't caues problems, but maybe it's cleaner without …

Reply:
should i do anything here?

Reply:
yes, please remove the `.pickle`

Reply:
done